fuel-client/assets/schema.sdl

```diff
@@ -282,6 +282,7 @@ type Mutation {
 	Submits transaction to the txpool
 	"""
 	submit(tx: HexString!): Transaction!
+	advanceBlock(advanceBy: U64): U64!
```

Comment:
I think other libraries use `mineBlock` for this but since that isn't really what's happening here I used `advance`, I can move it over to produce though as that also conveys the same message but more detailed